code/_onclick/ventcrawl.dm

```diff
@@ -0,0 +1,46 @@
+proc/isvent(var/atom/A)
+	if(istype(A,/obj/machinery/atmospherics/unary/vent_pump))
+		return 1
+	return 0
+
+/mob/living/carbon/slime/AltClickOn(var/atom/A)
```

Comment:
I had figured it would have been wasteful to have every mob check every time for vent crawling on alt click.

Also ventcrawl as a verb is only defined for specific mobs, and if you check their ventcrawl verbs some have specific conditions as well.